modules/stochastic_tools/examples/surrogates/gaussian_process/gold/gaussian_process_uniform_1D_tuned_out_cart_avg_0002.csv

```diff
@@ -1,101 +1,101 @@
 gauss_process_avg,gauss_process_avg_std
-300.54270648111,0.0026423121830013
```

Comment:
Do we need to update any of the figures/results in the GP example doc due to these changes?